file_paths.md

```diff
@@ -319,8 +319,8 @@ The package will install a service named `puppetserver`, create a
 
 This is a compatibility package using passenger to serve a ruby based puppetmaster.
 
-The package will install a service named `puppetmaster`, create a
-`puppet` user and group, and run the service as the `puppet` user.
+The package will install passenger in conjunction with Apache, create a
```

Comment:
I take that to mean the Apache install will create an `apache` service? If so, could you make that more explicit?